packages/README.md

```diff
@@ -1,3 +1,77 @@
 # Zarf Packages
 
 This folder contains packages maintained by the [Zarf team](https://github.com/defenseunicorns/zarf/graphs/contributors).  Some of these packages are used by `zarf init` for new cluster initialization.
+
+**Packages**
+- [distros](#distros)
+- [gitea](#gitea)
+- [logging-pgl](#logging-pgl)
+- [zarf-agent](#zarf-agent)
+- [zarf-registry](#zarf-registry)
+
+### Distros
+
+The distros package adds optional capabilities for spinning up and tearing down clusters.  Currently, the following distros are supported:
+
+- [EKS](https://aws.amazon.com/eks/) - Zarf deploys and tears down using the `eksctl` binary under the hood. See how it's done in the EKS package's [`zarf.yaml`](./distros/eks/zarf.yaml) and checkout the [EKS package's config](./distros/eks/eks.yaml) for more information.
+
+- [k3s](https://k3s.io/) - Zarf deploys and tears down using the `k3s` service under the hood. See how it's done in the k3s package's [`zarf.yaml`](./distros/k3s/common/zarf.yaml).
+
+
+#### Usage Examples  
+  
+**EKS**  - Create/Deploy EKS cluster. (requires `eksctl` credentials)
+
+```bash
+zarf package create packages/distros/eks -o build --confirm
+
+zarf package deploy build/zarf-package-distro-eks-multi-x.x.x.tar.zst --components=deploy-eks-cluster --set=CLUSTER_NAME='zarf-nightly-eks-e2e-test',INSTANCE_TYPE='t3.medium' --confirm
+```
+
+See the [nightly-eks test](../.github/workflows/nightly-eks.yml) for another example.
+
+**k3s** - Create/Deploy a k3s cluster.
```

Comment:
since we listed a pre-req for EKS, we should specify that this only works when you're running as a root user on a linux machine here

Reply:
@YrrepNoj  So it's `k3s`, correct? LMK if you want to see that note moved into the examples if it is good there.

Reply:
ah yes, definitely agree. Added more uniformity there:

- [Line 25](https://github.com/cmwylie19/zarf/blob/d9f335eb018ab9a42d0234547f29b2e509d6fcf6/packages/README.md?plain=1#L25)

- [Line 37](https://github.com/cmwylie19/zarf/blob/d9f335eb018ab9a42d0234547f29b2e509d6fcf6/packages/README.md?plain=1#L37)